types/pdfmake/index.d.ts

```diff
@@ -171,16 +181,24 @@ declare module 'pdfmake/build/pdfmake' {
     }
 
     interface TDocumentDefinitions {
-        info?: TDocumentInformation;
+        background?: (currentPage?: any, pageSize?: any) => any;
         compress?: boolean;
-        header?: TDocumentHeaderFooterFunction;
-        footer?: TDocumentHeaderFooterFunction;
         content: string | Content;
-        styles?: Style;
-        pageSize?: PageSize;
-        pageOrientation?: PageOrientation;
-        pageMargins?: Margins;
         defaultStyle?: Style;
+        footer?: TDocumentHeaderFooterFunction;
+        header?: TDocumentHeaderFooterFunction;
+        images?: Record<string, string>;
+        info?: TDocumentInformation;
+        pageBreakBefore?: (
+            currentNode?: any,
```

Comment:
The type of `currentNode` is available here: https://pdfmake.github.io/docs/document-definition-object/page/

Reply:
CurrentNode type has been added